### PR TITLE
data tables UI changes

### PR DESCRIPTION
**Pull request by Ran73Cohen**
Issue #330

### CODE REVIEW

File: src/components/common/TablePagination.vue [hunk 1]
```diff
@@ -1,7 +1,5 @@
 <template>
-  <div
-    class="d-flex justify-content-center align-items-center mt-3 font-size-14 font-w500"
-  >
+  <div v-if="pagesTotal > 0" class="undertable">
```
Comment (pingustar) [inline src/components/common/TablePagination.vue:2]: why remove the utility classes?

File: src/components/common/TablePagination.vue [hunk 2]
```diff
@@ -52,4 +52,13 @@ export default class TablePagination extends BaseComponent {
 }
 </script>
 
-<style lang="scss"></style>
+<style lang="scss">
+.undertable {
+  display: flex;
+  justify-content: center;
+  margin-block-start: 24px;
+  color: #6b7c93;
```
Comment (pingustar) [inline src/components/common/TablePagination.vue:60]: we should try to not hardcode these values.

**Comment by pingustar** [comment]
removing the tabs created inconsistency in design

image
image

File: src/components/common/TablePagination.vue (hunk 1 above)
Comment (Ran73Cohen) [reply src/components/common/TablePagination.vue:2]: I didn't know any better back then

**Review by pingustar — CHANGES_REQUESTED** [review]
lets update this to be more consistent in terms of code and design

**Review by pingustar — APPROVED** [review]
LGTM

**Pull request merged**